Chapter-3-microservice-extraction/Fitnet/Src/Reports/Fitnet.Reports/Fitnet.Reports.csproj

```diff
@@ -3,12 +3,13 @@
         <FrameworkReference Include="Microsoft.AspNetCore.App" />
     </ItemGroup>
     <ItemGroup>
-      <PackageReference Include="Dapper" Version="2.0.123" />
+      <PackageReference Include="Dapper" Version="2.1.15" />
       <PackageReference Include="evolutionaryarchitecture.fitnet.common.api" Version="1.1.5" />
```

Comment:
Would be great to update all packages to the newest version (can be done in the other PR)

Reply:
I'm going to do it in dotnet upgrade